internal/codegen/stencil.go

```diff
@@ -23,7 +23,7 @@ import (
 
 // NewStencil creates a new, fully initialized Stencil renderer function
 func NewStencil(m *configuration.ServiceManifest, mods []*modules.Module) *Stencil {
-	return &Stencil{m, extensions.NewHost(), mods}
+	return &Stencil{m, extensions.NewHost(), mods, true, make(map[string]interface{})}
```

Comment:
we should initialize with named struct fields - it's more readable especially with this many fields and results in no performance hits (compiler optimizes for you)

Reply:
Makes sense!